kubernetes/zenko/charts/mgob/templates/statefulset.yaml

```diff
@@ -0,0 +1,59 @@
+apiVersion: apps/v1
+kind: StatefulSet
```

Comment:
Why is this a StatefulSet but with a defined PVC? A StatefulSet generates its own PVC and I'm not entirely sure why there is a custom one.
https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/#stable-storage

Reply:
done in the latest commit.